plugins/setup.py

```diff
@@ -0,0 +1,18 @@
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
+setup(
+    version='0.0.1',
+    name='testgres-pg_probackup2',
```

Comment:
This setup.py is in the plugins folder and thus relates to all plugins in the folder (of which there can be many) but it's now written specifically for pg_probackup2. Wouldn't it be better to have here one common setup.py to install all plugins at once and have individual setup.py's inside each plugin folder to allow the user to install them selectively?